### ISSUE TITLE
MainTest does not respect java.home and JAVA_HOME

### ISSUE THREAD

**Issue by andreoss** [objectionary/eo#1107]
MainTest uses a default JDK, instead of the one used by maven

**Comment by 0pdd** [comment]
@andreoss the puzzle [#1109](https://github.com/objectionary/eo/issues/1109) is still not solved.

**Comment by yegor256** [comment]
@andreoss it's yours. As a general rule, please, work with an issue only after it's assigned to you. We need this for tracking purposes. Just ask in the issue (to me or @Graur) that you want to be the assignee and we will assign it to you.

**Comment by Graur** [comment]
#1109 is duplicate of #1008

**Comment by yegor256** [comment]
@graur please, resolve this one finally

**Comment by Graur** [comment]
@yegor256 As far as I understand it has been already solved [here](https://github.com/objectionary/eo/blob/85a167df8e6cfad774aa3bf6b46ef5a6708c8d14/eo-runtime/src/test/java/org/eolang/MainTest.java#L157-L164)

**Comment by 0pdd** [comment]
@andreoss the only puzzle [#1109](https://github.com/objectionary/eo/issues/1109) is solved here.
